README.md

```diff
@@ -1,2 +1,37 @@
 # hw01
 Repository for homework 01
+
+# Hi, my name is Rachel Kim
+
+I'm a fourth year sociology major in the college. I'm also a history minor. Broadly, I study: 
+
+* race
+* tech
+* cities
+* and the American South.
+
+### Where I'm From
+I'm originally from Alpharetta, Georgia, which is a suburb about thirty minutes northeast of Atlanta. 
+
+There's nothing notable about the area, except for the fact that some Real Housewives of Atlanta used to live around there.
+
+![Like Kim, whose daughter briefly went to my high school.](https://media.giphy.com/media/T8mmXYupagKfC/giphy.gif)
+
+### What I Do
+I am the education editor of the *South Side Weekly*, where I read, report, and edit pieces about Chicago Public Schools and South Side students, teachers, and families. It's the **best damn paper in the city of Chicago**, if I do say so myself.
+
+Shameless plug: our [2018 Best of the South Side issue](https://southsideweekly.com/best-of-the-south-side-2018/).
+
+I'm also a barista at ExLibris, which is in the Reg.
+
+### Coding Experience
+None!
+But if it counts, I did some rudimentary html design on [Neopets](http://www.neopets.com//)—the website where I spent all of my hours as a kid. Mostly I just used the "marquee" command as often as possible. I also just learned that it works in R. 
+
+<marquee> **This is incredibly exciting.** </marquee>
```

Comment:
Nice extension. That's HTML code, right? In a standard `html_document` that would show up correctly. Unfortunately GitHub limits the use of HTML code in Markdown documents to prevent individuals from adding malicious code to their site